chain-addr/src/lib.rs

```diff
@@ -24,11 +24,17 @@ use std::string::ToString;
 // temporary re-use just to define
 use cardano::redeem::{self, PublicKey};
 
-use chain_core::property;
+use chain_core::property::{self, Serialize as PropertySerialize};
```

Comment:
Nitpick: I'd just use `property::Serialize` like everywhere else, the whole-module import is there anyway.

Reply:
I need to import the trait so I can use its methods on structs implementing it. Without this import I would have to use UFC.